# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/tests/non262/fields/field_types.js
# Commit: d96f98f974e0
# Full Hash: d96f98f974e087224e0a1ef9be67e2fc63397d8b
# Author: Ashley Hauck <khyperia@mozilla.com>
# Date: 2019-03-21 04:34:45
# Regressor Bug: 1535166
# File Overlap Count: 1
# Description:
#   Bug 1535166 - Implement computed field names. r=jorendorff
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D23408
# ==============================================================================

diff -r c786b72a9d8e -r d96f98f974e0 js/src/tests/non262/fields/field_types.js
--- a/js/src/tests/non262/fields/field_types.js	Wed Mar 20 15:50:30 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,25 +0,0 @@
-// * * * THIS TEST IS DISABLED - Fields are not fully implemented yet
-
-class C {
-    [Math.sqrt(4)];
-    [Math.sqrt(8)] = 5 + 2;
-    "hi";
-    "bye" = {};
-    2 = 2;
-    0x101 = 2;
-    0o101 = 2;
-    0b101 = 2;
-    NaN = 0; // actually the field called "NaN", not the number
-    Infinity = 50; // actually the field called "Infinity", not the number
-    // all the keywords below are proper fields (?!?)
-    with = 0;
-    //static = 0; // doesn't work yet
-    async = 0;
-    get = 0;
-    set = 0;
-    export = 0;
-    function = 0;
-}
-
-if (typeof reportCompare === "function")
-  reportCompare(true, true);